src/orquestra/quantum/api/wavefunction_simulator.py

```diff
@@ -0,0 +1,101 @@
+################################################################################
+# © Copyright 2022 Zapata Computing Inc.
+################################################################################
+
+import numpy as np
+from typing import Protocol, Optional
+from ..circuits import Circuit, split_circuit, Operation, GateOperation
+from ..measurements import Measurements
+from .circuit_runner import CircuitRunner, BaseCircuitRunner
+from abc import abstractmethod
+from ..wavefunction import sample_from_wavefunction, Wavefunction
+from ..operators import PauliRepresentation, get_expectation_value
+from ..distributions import (
+    MeasurementOutcomeDistribution,
+    create_bitstring_distribution_from_probability_distribution,
+)
+
+# TODO: Statevector definition should be moved to typing (I think?)
```

Comment:
Could you fix this TODO (either fix or remove, whatever you find better)

Reply:
Fixed in [a2fd221](https://github.com/zapatacomputing/orquestra-quantum/pull/66/commits/a2fd221be3fe0293841ffca2d4efd5c6ad20f32c)